starboard/android/shared/egl_swap_buffers.cc

```diff
@@ -16,14 +16,25 @@
 #include <GLES2/gl2.h>
 
 #include "starboard/android/shared/video_window.h"
+#include "starboard/common/atomic.h"
```

Comment:
Note we are switching to std <atomic> everywhere, but if this doesn't land back on trunk it's fine to leave it as is.